modules/apps/adaptive-media/adaptive-media-image-impl/src/test/java/com/liferay/adaptive/media/image/internal/util/AMImageSerializerImplTest.java

```diff
@@ -193,4 +194,6 @@ public void testSerializeAdaptiveMediaWithEmptyAttributes()
 		Assert.assertEquals(0, attributesJSONObject.length());
 	}
 
+	private static AMImageSerializer _amImageSerializer;
```

Comment:
@julschong It does not need to be static here, as your setup class is for each test method rather than doing once for all test methods.